internalshared/configutil/http_response_headers.go

```diff
@@ -0,0 +1,186 @@
+package configutil
+
+import (
+    "fmt"
+    "net/textproto"
+    "strconv"
+    "strings"
+)
+
+var DefaultHeaderNames = []string {
+    "Content-Security-Policy",
+    "X-XSS-Protection",
+    "X-Frame-Options",
+    "X-Content-Type-Options",
+    "Strict-Transport-Security",
+    "Content-Type",
+}
+
+var ValidCustomStatusCodeCollection = []string {
+    "default",
+    "1xx",
+    "2xx",
+    "3xx",
+    "4xx",
+    "5xx",
+}
+
+const (
+    contentSecurityPolicy = "default-src 'none';   connect-src 'self';  img-src 'self' data:; script-src 'self'; style-src 'unsafe-inline' 'self'; form-action  'none'; frame-ancestors 'none'; font-src 'self'"
+    xXssProtection = "1; mode=block"
+    xFrameOptions = "Deny"
+    xContentTypeOptions = "nosniff"
+    strictTransportSecurity = "max-age=31536000; includeSubDomains"
```

Comment:
This header won't be honored by browsers on HTTP connections but will definitely please the scanners. I think the default is fine as-is

Reply:
Yes, that is one of the intentions of including that here :)